test/Xpring/xpring-client.test.ts

```diff
@@ -20,22 +24,23 @@ const fakeAccountExistsResult = true
 const fakeRawTransactionStatus = new RawTransactionStatus(
   true,
   'tesSUCCESS',
-  10,
+  fakeLastLedgerSequenceValue,
   true,
 )
 const fakePaymentHistoryValue = []
 const fakeGetPaymentValue = testXRPTransaction
+/* eslint-enable @typescript-eslint/no-magic-numbers, @typescript-eslint/naming-convention */
 
-/** An amount to send. */
+// An amount to send
```

Comment:
I guess I could have left the old comments here?

In VSCode, if you use JSDoc style syntax over a variable, it shows up in the tooltip when you hover over that variable which is super neat. But I didn't feel like that was necessarily the intention here?

Reply:
That's fine. I tend to use /* */ on static vars but it's just a random convention I have in my head.

Reply:
Yeah, I have random conventions in my head as well for when to use `//` line comments and when to use `/* */` multi-line comments (even when it's on a single line).